RFTrain/src/model.py

```diff
@@ -0,0 +1,80 @@
+import os
+from shutil import copyfile
+
+import joblib
+from sklearn.metrics import precision_score
+from sklearn.model_selection import train_test_split, cross_val_score, KFold
+
+from . model_factory import ModelFactory
+from . utils import set_train_path
+
+
+class Model:
+    def __init__(self, config):
+        self.random_state = config.get('random_state')
+        self.model_type = config.get('model_type')
+        self.model_config_path = config.get('model_config_path')
+        self.model_train_path = set_train_path(config.get('model_train_dir'))
+        self._model = ModelFactory(self.model_config_path, self.model_type).make_model()
+
+    def set_model(self, model_config_path, model_type):
+        self._model = ModelFactory(model_config_path, model_type).make_model()
+
+    def predict(self, features):
+        return self._model.predict(features)
+
+    def save_model(self):
+        copyfile(src=self.model_config_path, dst=os.path.join(self.model_train_path, 'training_settings.ini'))
+        return joblib.dump(self._model, os.path.join(self.model_train_path, 'trained.sav'))
+
+
+class OCCModel(Model):
+    def fit(self, neutrals):
+        return self._model.fit(neutrals)
+
+    def train(self, neutral, positive):
+        self.fit(neutral)
+        pos_accuracy = sum(self.predict(positive) == -1) / len(positive)
+        with open(os.path.join(self.model_train_path, "training_results.txt"), 'a') as f:
+            to_print = "Training error fraction: {}\n" \
+                       "Accuracy on positives: {}\n".format(self._model.nu, pos_accuracy)
```

Comment:
I believe writing as `f"{pos_accuracy}..." will be much simpler.

Reply:
And everywhere below.

Reply:
changed to your preference